nixos/modules/services/security/authelia.nix

```diff
@@ -308,7 +308,7 @@ in
         {
           description = "Authelia authentication and authorization server";
           wantedBy = [ "multi-user.target" ];
-          after = [ "network.target" ];
+          after = [ "network-online.target" ]; # Checks SMTP notifier creds during startup
```

Comment:
For `network-online.target`, the unit also needs to depend on it:
```suggestion
          after = [ "network-online.target" ]; # Checks SMTP notifier creds during startup
          wants = [ "network-online.target" ];
```

Reply:
Manually applied, thanks!